frontend/packages/schema-editor/src/components/SchemaInspector/CustomProperties.tsx

```diff
@@ -1,12 +1,18 @@
 import React from 'react';
-import { Button, Checkbox, FieldSet, HelpText, TextField } from '@digdir/design-system-react';
+import {
+  Button,
+  LegacyCheckbox,
```

Comment:
Jus a suggestion, but I'm wondering if it would not be better to import Legacy components using aliases to avoid renaming our tags (<Checkbox, <RadioButton, etc) 🤔 
```suggestion
  LegacyCheckbox as Checkbox,
```

Reply:
I agree with @mlqn that casting/alias as Checkbox is easier, but I may prefer renaming all the tags to Legacy to make it clear that the code is using is the Legacy version of the component. Then developers do not need to see/catch the alias casting within the import statement to determine whether this component is the legacy or not. Just my thoughts 🙈